packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapPolylineController.h

```diff
@@ -13,6 +13,7 @@
 - (void)setStrokeWidth:(CGFloat)width;
 - (void)setPoints:(NSArray<CLLocation*>*)points;
 - (void)setZIndex:(int)zIndex;
+- (void)setGeoDesic:(BOOL)isGeodesic;
```

Comment:
Geodesic is a [word](https://en.wikipedia.org/wiki/Geodesic) in itself (not GeoDesic), please make this `setGeodesic`